porch/func/go.mod

```diff
@@ -3,33 +3,60 @@ module github.com/GoogleContainerTools/kpt/porch/func
 go 1.17
 
 require (
+	github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/apply-replacements v0.0.0-20220316042125-d3eb289008bc
+	github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/starlark v0.3.0
+	github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20220316202203-f9115a993ebd
 	google.golang.org/grpc v1.44.0
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/klog/v2 v2.40.1
 	sigs.k8s.io/kustomize/kyaml v0.13.3
 )
 
+replace github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/starlark v0.3.0 => github.com/mengqiy/kpt-functions-catalog/functions/go/starlark v0.0.0-20220316204754-e2611d41524d
```

Comment:
Will remove this after https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/778 merges

Reply:
ack, sounds good.